lib/server/operations/version.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operations
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/hyperledger/fabric/common/flogging"
+)
+
+// VersionInfoHandler stores the current version information of the CA
+type VersionInfoHandler struct {
+	Version string `json:"Version,omitempty"`
+}
+
+func (m *VersionInfoHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
+	switch req.Method {
+	case http.MethodGet:
+		m.sendResponse(resp, http.StatusOK, m)
+	default:
+		err := fmt.Errorf("invalid request method: %s", req.Method)
+		m.sendResponse(resp, http.StatusBadRequest, err)
+	}
+}
+
+type errorResponse struct {
+	Error string `json:"Error"`
+}
+
+func (m *VersionInfoHandler) sendResponse(resp http.ResponseWriter, code int, payload interface{}) {
+	if err, ok := payload.(error); ok {
+		payload = &errorResponse{Error: err.Error()}
+	}
+	js, err := json.Marshal(payload)
+	if err != nil {
+		logger := flogging.MustGetLogger("operations.runner")
+		logger.Errorw("failed to encode payload", "error", err)
+		resp.WriteHeader(http.StatusInternalServerError)
+	} else {
+		resp.WriteHeader(code)
+		resp.Header().Set("Content-Type", "application/json")
+		resp.Write(js)
+	}
+}
```

Comment:
Given you use 
```
s.mux.HandleFunc("/version", func(w http.ResponseWriter, _ *http.Request) 
```
in initializeVersionInfoHandler(), seems like you actually only need the type and no member functions?

Reply:
Ah, thanks, bad copy from Fabric where there exists a better handler mechanism

Reply:
Fixed to use the actual Handler I created.